src/translations/en.json

```
@@ -1493,7 +1494,9 @@
                 "metric_example": "Celsius, kilograms",
                 "find_currency_value": "Find your value",
                 "save_button": "Save",
-                "currency": "Currency"
+                "currency": "Currency",
+                "edit_location": "Edit location",
+                "edit_location_description": "Location can be editted on the zones page"
```

"editted" feels weird. Maybe this:
```suggestion
                "edit_location_description": "Location can be changed in zone settings"
```

Or just "changed" instead of "editted"